Source/UETorch/Private/UETorch.cpp

```diff
@@ -973,4 +973,65 @@ extern "C" bool DestroyActor(AActor* object) {
 	}
 	object->K2_DestroyActor();
 	return true;
+}
+
+/**
+ * Functions to control pawns
+ */
+
+extern "C" bool SimpleMoveToLocation(UObject* _this, AActor* object, float x, float y, float z) {
```

Comment:
Hmm. Well, I would first suggest a better name, e.g. `NavigatePawnToLocation(UObject* _this, AActor* pawn, float x, float y, float z)`, and maybe a comment explaining what it does (your response above is perfect).

Second, do you think this function is appropriate for UETorch? I.e. Is this a general function that you think people will use? Or is this specific to your particular application?